src/rules/prefer-const.md

```diff
@@ -94,7 +93,7 @@ console.log(a);
 
 // `i` gets a new binding each iteration
 for (const i in [1, 2, 3]) {
-  console.log(i);
+  console.log(a);
```

Comment:
Weird, in the original document it used to be `a` and now it's changed to `i`, but this change seems to be doing the opposite.